### DISCUSSION TITLE
How to read epub files

### DISCUSSION THREAD

**Discussion by claudia-sunhy** [ken107/read-aloud#409] [Q&A] [answered]
I've downloaded the EPUBReader extension. But ReadAloud reported "Cannot access a chrome-extension:// URL of different extension". How can I get those two work together?

**Comment by ken107** [comment]
Could you provide the URL? And what browser are you using?

**Comment by claudia-sunhy** [comment, marked as answer]
No worries. It looks like I had initially downloaded the wrong EPUB reader extension. The link "epubread.com" keeps directing me to the same extension I already had installed which is called EpubPup. Once I uninstalled the incorrect extension, the link successfully led me to the right one, and it’s working perfectly now. Thank you.
